force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -204,9 +326,13 @@ public with sharing class FlowControlLoopsCollections {
         List<Integer> numberList = new List<Integer>{1, 2, 3, 4, 5}; //DO NOT CHANGE
 
         // Remove the 3rd element (number 3) from the list
+        Integer index = numberList.indexOf(3);
+        System.debug(index);
+        numberList.Remove(index);
```

Comment:
Solid logic here, the requirement was a little fuzzy and you handled it well

Reply:
Thank you very much!